.jenkins/pytorch/build.sh

```diff
@@ -115,6 +115,12 @@ if [[ "$BUILD_ENVIRONMENT" == *trusty-py3.6-gcc5.4* ]]; then
   export DEBUG=1
 fi
 
+# Patch required to build xla
+if [[ "${JOB_BASE_NAME}" == *xla* ]]; then
+  git clone --recursive https://github.com/pytorch/xla.git
+  patch -p1 < xla/pytorch.patch
```

Comment:
topkek lol. Upstream when?!

Reply:
;) Should be soon, https://github.com/pytorch/xla/blob/master/pytorch.patch they're working on making `XLATensor` a real `Tensor` :D